### ISSUE TITLE
No audio for AC-3 in MPEG-TS

### ISSUE THREAD

**Issue by stari4ek** [google/ExoPlayer#1108]
* checked with master (1.5.3) and dev (f4c10dc560fbe46a4fee8bea9b6d7079d3e03ed0)
* problematic video file: https://docs.google.com/uc?id=0B4_pcf38WwzYdlhlWjlSeWdLMDQ&export=download
* originally this is live stream coming with multicast/UDP, but it doesn't work with local file also
* checked with DemoPlayer
```c++
     // Samples.java
      new Sample("No audio: TS/h264/AC3",
                 "asset:///nickelodian_hd.ts", PlayerActivity.TYPE_OTHER),
     // ExtractorRendererBuilder.java: set proper TsExtractor. see issue #982
         ExtractorSampleSource sampleSource = new ExtractorSampleSource(uri, dataSource, allocator,
        BUFFER_SEGMENT_COUNT * BUFFER_SEGMENT_SIZE,
        new TsExtractor(new PtsTimestampAdjuster(0), false));
```
I can provide ready-to-test branch - didn't want to put video blob (20Mb) to git
* file is MPEG_TS with h264 video and 2 AC-3 audio tracks
* checked on NVidia Shield and Simulator x86 API 23

Video playback starts normally but player does not identify audio tracks

**Comment by andrewlewis** [comment]
This should be fixed by 6eea4bdbc34dea86a5bc81991163ca26761cae83. Please reopen if it still doesn't work as expected. Thanks!

**Comment by stari4ek** [comment]
* on NVidia Shield: works well
* on Samsung Note 4 (N910U) doesn't. But probably AC-3 isn't supported by device
